src/main/drivers/max7456_symbols.h

```diff
@@ -34,49 +29,49 @@
 // Degrees Icon for HEADING/DIRECTION HOME
 #define SYM_DEGREES 0xBD
 
-// Direction arrows
-#define SYM_ARROW_UP    0x60
+// Direction arrows 
+#define SYM_ARROW_SOUTH 0x60
```

Comment:
Still, UP, DOWN, LEFT, RIGHT is more logical, since we are dealing with the image on the screen, not actual orientation.